### PR TITLE
JBTIS-451 - second attempt - shorter title and linked installation ins…

### PR DESCRIPTION
**Pull request by pleacu**
…tructions

### CODE REVIEW

**Comment by pleacu** [comment]
@maxandersen - here's a modified blog - wdyt?

File: blog/JBTIS-422-Final.adoc [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+:page-layout: blog
+:page-author: pleacu
+:page-date: 2015-06-30
+:page-tags: [release, jbosstools, devstudio, jbosscentral]
+
+JBoss Fuse Tooling plus an updated Teiid Designer and Early Access versions of the Fuse SAP Tool Suite, Fuse Data Transformation tooling and SwitchYard (Luna/ JBDS 8).
+
+image::/blog/images/jbosstools-jbdevstudio-blog-header.png[]
+
+*JBoss Tools Integration Stack 4.2.2.Final / JBoss Developer Studio Integration Stack 8.0.2.GA*
```
Comment (maxandersen) [inline blog/JBTIS-422-Final.adoc]: changed this to use proper image caption/title.

File: blog/JBTIS-422-Final.adoc [hunk 2]
```diff
@@ -0,0 +1,108 @@
+= Integration tooling for Fuse + Early Access Data Trans and SAP support
+:page-layout: blog
+:page-author: pleacu
+:page-date: 2015-06-30
+:page-tags: [release, jbosstools, devstudio, jbosscentral]
+
+JBoss Fuse Tooling plus an updated Teiid Designer and Early Access versions of the Fuse SAP Tool Suite, Fuse Data Transformation tooling and SwitchYard (Luna/ JBDS 8).
```
Comment (maxandersen) [inline blog/JBTIS-422-Final.adoc]: JBDS is the jira key, JBoss Developer Studio is the name to use in blogs etc.

File: blog/JBTIS-422-Final.adoc [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+* All of the Business Process and Rules Development plugins, plus...
+* link:/features/apachecamel.html[Fuse Apache Camel Tooling] - A graphical tool for integrating software components that works with Apache ServiceMix, Apache ActiveMQ, Apache Camel and the FuseSource distributions.
+* link:/features/switchyard.html[SwitchYard] - A lightweight service delivery framework providing full lifecycle support for developing, deploying, and managing service-oriented applications.
+
+=== SOA 5.x Development
+
+* link:http://www.jboss.org/jbossesb[JBoss ESB] - An enterprise service bus for connecting enterprise applications and services.
+* link:http://docs.jboss.com/jbpm/v3.2/userguide/html_single/[jBPM3] - A flexible Business Process Management (BPM) Suite - JBoss Enterprise SOA Platform 5.3.x compatible version.
+
+*All of these components have been verified to work with the same dependencies as JBoss Tools 4.2 and Developer Studio 8.*
```
Comment (maxandersen) [inline blog/JBTIS-422-Final.adoc]: could change this to a NOTE admonition instead  of just bold.

File: downloads/is_installation.adoc [hunk 4]
```diff
@@ -0,0 +1,76 @@
+= Installation
+:page-layout: project
+
+== If you already have JBDSIS 8.1.0 installed...
+
+Simply start jbdevstudio or eclipse-with-jbds, then:
```
Comment (maxandersen) [inline downloads/is_installation.adoc]: this is all very version specific. lets update it to be bit more useful for future updates.

**Comment by maxandersen** [comment]
i've pushed a version that does not use JBDS, used a bit more specific formatting rather than just bold and "refactored" the is_documentation into something less version specific.

File: downloads/is_installation.adoc [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+image::/blog/images/jbtis/jbtis-422-b6.png[JBoss Central Early Access - JBoss Tools Integration Stack]
+
+The standard p2 installer is available for JBoss Tools Integration Stack.  Simply start eclipse-with-jbt, then:
+
+[source]
+----
+ Help > Install New Software...
+ Add...
+ - use this for 'Location:' for the released-components integration stack:
+   https://devstudio.redhat.com/updates/8.0/integration-stack/
```
Comment (maxandersen) [inline downloads/is_installation.adoc]: these are not the public urls we use https://devstudio.redhat.com/updates/8.0 should be sufficient

**Comment by maxandersen** [comment]
merged in version where the mentioned issues are fixed. 

open PR/jira if you find something bad, but now at the blog is at least out.

**Pull request closed without merging**